package.json

```diff
@@ -59,6 +59,13 @@
   "devDependencies": {
     "babel-plugin-styled-components": "^1.5.1",
     "babel-plugin-transform-object-assign": "^6.22.0",
-    "prettier": "~1.17.0"
+    "husky": "^2.3.0",
+    "prettier": "~1.17.0",
```

Comment:
Should this be `^1.17.0` again? Since .pre-commit-config.yaml was deleted so no need to match the version in there anymore.

Reply:
@jorgeorpinel Yes, you are right. I will change it.